### PR TITLE
Implement Update Functionality

### LINKED ISSUE #15: As a delivery assistant, I want to be able to edit the stocks in the inventory in bulk, so that I can save time and do deliveries more efficiently.

### LINKED ISSUE #25: As a tech savvy warehouse manager, I want to be able to change the information of my existing stock in the application so that I can keep my existing inventories updated.

### LINKED ISSUE #81: Add Update feature to Warenager

### LINKED ISSUE #82: Implement increment and decrement stock quantity as part of update feature

### LINKED ISSUE #83: Implement renew stock quantity as part of update feature

### LINKED ISSUE #84: Implement update stock name as part of update feature

### LINKED ISSUE #85: Implement update stock location as part of update feature

### LINKED ISSUE #86: Implement update stock source as part of update feature

### PR DESCRIPTION
**Pull request by AudreyFelicio**

### CODE REVIEW

**Comment by Ashley-Lau** [comment]
update sn/fairprice1 iq/1000 sn/fairprice2 seems to be incrementing both stock by 1000. Is this the expected behavior?

**Comment by AudreyFelicio** [comment]
> update sn/fairprice1 iq/1000 sn/fairprice2 seems to be incrementing both stock by 1000. Is this the expected behavior?

Yes, since now we allow to update multiple stocks at once.

**Review by kaichaoang — COMMENTED** [review]
LGTM!

File: src/main/java/seedu/stock/logic/commands/UpdateCommand.java [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+        requireNonNull(model);
+        List<Stock> lastShownStocks = model.getFilteredStockList();
+
+        List<SerialNumber> indexes = updateStockDescriptor.getSerialNumbers();
+        List<String> serials = indexes.stream().map((serial) -> serial.getSerialNumberAsString().trim())
+                .collect(Collectors.toCollection(ArrayList::new));
+        List<Stock> stocksToUpdate = new ArrayList<>();
+        List<Stock> updatedStocks = new ArrayList<>();
+
+        // FInd stocks to be updated
```
Comment (Ashley-Lau) [inline src/main/java/seedu/stock/logic/commands/UpdateCommand.java:92]: just a small typo here I chanced upon

**Review by Ashley-Lau — APPROVED** [review]
LGTM

**Pull request merged**